NuCivic/dkan
# Fix dkan_workflow_permissions override and role assign js

Issue: CIVIC-4442

## Description

- Dkan Workflow Permissions overridden OOB
- dkan_workflow_permissions.js does not work for site managers
- Add tests for role pairing javascript

## How to reproduce

1.  Enable dkan_workflow
2. View feature overrides for dkan_workflow_permissions
3. log in as a site manager
4. Go to People > Create User
5. Select one of the workflow roles, confirm it does not also select a standard paired role

## QA Tests

- [x] Confirm dkan_workflow_permissions is not overridden when enabled
- [ ] Confirm that when logged in as a site manager, assigning a workflow role to a user automatically assigns the paired standard role.
